src/test/java/net/imglib2/mesh/alg/StupidMesh.java

```diff
@@ -0,0 +1,151 @@
+/*-
+ * #%L
+ * Volume rendering of bdv datasets
+ * %%
+ * Copyright (C) 2018 - 2021 Tobias Pietzsch
+ * %%
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ * #L%
+ */
+package net.imglib2.mesh.alg;
+
+import static com.jogamp.opengl.GL.GL_FLOAT;
+import static com.jogamp.opengl.GL.GL_UNSIGNED_INT;
+
+import java.awt.Color;
+import java.nio.FloatBuffer;
+import java.nio.IntBuffer;
+
+import org.joml.Matrix3f;
+import org.joml.Matrix4f;
+import org.joml.Matrix4fc;
+
+import com.jogamp.opengl.GL;
+import com.jogamp.opengl.GL3;
+
+import bvv.core.backend.jogl.JoglGpuContext;
+import bvv.core.shadergen.DefaultShader;
+import bvv.core.shadergen.Shader;
+import bvv.core.shadergen.generate.Segment;
+import bvv.core.shadergen.generate.SegmentTemplate;
+import net.imglib2.mesh.obj.nio.BufferMesh;
+
+public class StupidMesh
```

Comment:
As entertaining as I find the name `StupidMesh`, I would suggest not to name it in this way, because it does not in fact implement the `Mesh` interface, so the name is confusing. It's more of a container for a mesh plus rendering details and logic for drawing it inside a JOGL GL3. Right? So then... how about something like `GLMeshView`?